framework/src/auxkernels/AuxKernel.C

```diff
@@ -179,6 +179,16 @@ AuxKernelTempl<ComputeValueType>::AuxKernelTempl(const InputParameters & paramet
     paramError("variable",
                "Variable family " + Moose::stringify(type.family) + " is not supported at order " +
                    Moose::stringify(type.order) + " by the AuxKernel system.");
+
+  // The AuxKernel computation loops rely on quadrature point evaluations. Up until the introduction
+  // of FVAuxKernels we will require quadrature point evaluations for the variables general
+  // AuxKernels act on.
```

Comment:
Based on this comment, I'm not convinced of the need for calling `requireQpComputations()` since the aux kernel will never call `_u[_qp]` itself

Reply:
This is needed because in the AuxKernel we have this:
```
_var.setDofValues(_local_sol);
```
which operates both on `_vector_tags_dof_u` and `_vector_tag_u` which are resized in `MooseVariableDataFV`
```
template <typename OutputType>
void
MooseVariableDataFV<OutputType>::prepareIC()
{
  // TODO: implement this function
  initDofIndices();
  _vector_tags_dof_u[_solution_tag].resize(_dof_indices.size());

  mooseAssert(_qrule, "We should have a non-null qrule");
  const auto nqp = _qrule->n_points();
  _vector_tag_u[_solution_tag].resize(nqp);
}
```
but in the FV initial condition loop we don't want to use the data variables. 

Now that I think about it, I can just call the `prepareIC` functions from the loop and just have it as an empty function override in the Linear variable which is going to come.

Reply:
Ok that does the trick too. Will go with that since it is much closer to the original approach.

Reply:
Yeah never mind, I don't know what I was thinking. We have a loop over `ElemInfos` here without resetting anything in the assembly/data members. So the `prepareIC` approach does not work.

Reply:
Well, in a hacky way it still seems to work. The data members are not used, they are just resized so that they can be populated by the auxkernels.